test/spec/modules/conversantAnalyticsAdapter_spec.js

```diff
@@ -45,25 +48,33 @@ describe('Conversant analytics adapter tests', function() {
     };
     sandbox.stub(prebidGlobal, 'getGlobal').returns(getGlobalStub); // getGlobal does not seem to be available in testing so need to mock it
     clock = sandbox.useFakeTimers(DATESTAMP); // to use sinon fake timers they MUST be created before the interval/timeout is created in the code you are testing.
+
+    logInfoStub = sandbox.stub(utils, 'logInfo');/* .callsFake((arg, arg1, arg2) => { //debugging stuff
+      console.log(arg);
+      if (arg1) console.log(arg1);
+      if (arg2) console.log(arg2);
+    }); */
+
```

Comment:
Remove commented out section?

Reply:
Yeah, there's an argument to be made to do so.  I guess I hate having to figure out how I did this each time I need to debug when I work on this.